client/containers/DashboardOverview/overviewRows/overviewRowSkeleton.scss

```diff
@@ -67,6 +67,13 @@ $mobile-viewport-cutoff: 750px;
 		}
 		.byline {
 			font-style: italic;
+			.byline-icon {
```

Comment:
Let's move the repeated CSS related to the `.byline-icon` into the purview of the `Byline` component. I think this'll mean creating a new `byline.scss` file and importing it from `Byline.tsx`.